auth/store.go

```diff
@@ -1152,20 +1152,24 @@ func (as *authStore) Revision() uint64 {
 }
 
 func (as *authStore) AuthInfoFromTLS(ctx context.Context) (ai *AuthInfo) {
+	ai = &AuthInfo{}
```

Comment:
Let's keep it `nil`. `processInternalRaftRequestOnce` updates this if `authInfo != nil`.

Reply:
thanks fixed.